packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -274,12 +288,26 @@ export abstract class QueueProcessingServiceBase extends Construct {
   /**
    * The scaling interval for autoscaling based off an SQS Queue size.
    */
+
```

Comment:
```suggestion
```
The new line is needed only between different props, comments shouldn't be separated from the related parameter.

Reply:
Thanks Updated.